flask_common/utils/legacy.py

```diff
@@ -697,3 +694,39 @@ def __getitem__(self, key):
         if self.data is None:
             self.data = list(self.f())
         return self.data[key]
+
+
+__all__ = [
```

Comment:
I prefer `__all__` to be at the top of the file after imports but don't care too much.